docs/quick_start.rst

```diff
@@ -6,27 +6,33 @@ Listing templates and projects
 
 Temple manages projects that are started from templates
 (specifically, `cookiecutter`_ templates).
-In order to see what templates are available for use within your Github org, do::
+In order to see what templates are available for use, do::
 
-    temple ls <github_org_name>
+    temple ls <forge>
 
-This will list all of the paths of templates that are available. Doing::
+This will list all of the paths of templates that are available under a particular git forge.
+A git forge can be:
 
-    temple ls <github_org_name> -l
+1. A Github user or organization, such as ``github.com/OrganizationName``
+2. A Gitlab group, such as ``gitlab.com/Group`` or ``gitlab.com/Nested/Group``
```

Comment:
Does this work for a GitLab user as well, for example https://gitlab.com/users/alysivji/projects?

Reply:
I just checked and it does not work. I'm parsing the URL and assuming the group is right after `gitlab.com` at the moment. I do think this is a valid thing to support though, so I can see if it's trivial. If it isn't, I would suggest adding it as an issue that can be tracked